.sync.yml

```diff
@@ -0,0 +1,12 @@
+:global:
```

Comment:
Please put modulesync-managed config in place for this PR (golden tests GitHub actions etc.). Otherwise, it's quite possible that you'll commit stale golden test outputs, and I'm not going to manually verify that the committed golden test outputs match the actual output of compiling the component.

I would just move this config to a separate PR and then make sure you run and merge the sync for this repo. You can then rebase this PR on the latest master branch.

Reply:
I was having doubts as to how to move on with modulesync. Ok I will add that file in a new PR.

Reply:
Let's try another round of review :), Hopefully this time around everything clicks